src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: {
+      reducer: (state, action) => {
+        state.books.push(action.payload);
+      },
+      prepare: (name, author) => ({
+        payload: {
+          name,
+          author,
+        },
+      }),
+    },
+    removeBook: (state, action) => {
+      const { id } = action.payload;
+      state.books.filter((book) => book.id !== id);
+    },
+  },
+});
+
+export default booksSlice.reducer;
```

Comment:
- [ ] Dear @Akorede3133, Alongside the `booksSlice.reducer;` we need to export the `{ addBook, removeBook }` form `booksSlice.actions`. Please export these two actions.